### PR TITLE
Removed references to nested content

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

Removed references to nested content from v10, v12, and v13. Let me know if I should do it for v14 as well.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

**Comment by alina-tincas** [comment]
hmm, I think the main article should be left in the docs for v10, 12 and 13 as it can still be used even is if it is discontinued. It is only from v14 that it is completely removed.

**Comment by alina-tincas** [comment]
and also leave the release notes as they are with nested content if they include something about nested content

**Comment by alina-tincas** [comment]
ok, found it in CMS repo:
image

so the warning is "Nested content is obsolete, will be removed in V13"

**Comment by alina-tincas** [comment]
So my recommendation is:
1. Leave the release notes from deploy as they are with nested content included as am sure there is a reason why it was mentioned there.
2. on v10 and v12 leave the main article **umbraco-cms\fundamentals\backoffice\property-editors\built-in-umbraco-property-editors\nested-content.md** in the summary so that people are aware that it has been discontinued
3. on v13 and v14 you can remove everything about nested content 👍

**Comment by eshanrnh** [comment]
@alina-tincas Thanks for your review 🙌I've incorporated the comments. Let me know what you think 🙂

File: 12/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/block-list-editor.md [hunk 1]
```diff
@@ -7,9 +7,7 @@
 **Block List** is a list editing property editor, using [Element Types](../../../../data/defining-content/#element-types) to define the list item schema.
 
 {% hint style="info" %}
-A default Umbraco CMS installation does not ship with a defined _Data Type_ using the Block List editor. In order to start using the property, follow the steps outlined below.
-
-[Click here for an overview with a worked example and references back to the relevant documentation.](https://umbraco.com/blog/deep-dive-the-block-list-editor/)
+The _Block List_ replaces the obsolete _Nested Content_ control.
```
Comment (alina-tincas) [inline 12/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/block-list-editor.md:10]:
```suggestion
The _Block List_ replaces the obsolete _Nested Content_ editor.
```

File: 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/block-list-editor.md [hunk 2]
```diff
@@ -7,9 +7,7 @@
 **Block List** is a list editing property editor, using [Element Types](../../../../data/defining-content/#element-types) to define the list item schema.
 
 {% hint style="info" %}
-A default Umbraco CMS installation does not ship with a defined _Data Type_ using the Block List editor. In order to start using the property, follow the steps outlined below.
-
-[Click here for an overview with a worked example and references back to the relevant documentation.](https://umbraco.com/blog/deep-dive-the-block-list-editor/)
+The _Block List_ replaces the obsolete _Nested Content_ control.
```
Comment (alina-tincas) [inline 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/block-list-editor.md:10]:
```suggestion
The _Block List_ replaces the obsolete _Nested Content_ editor.
```

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
LGTM 💪 🙌 #h5YR

**Pull request merged**